core/domain/story_services.py

```diff
@@ -0,0 +1,318 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Commands that can be used to operate on stories.
+
+All functions here should be agnostic of how StoryModel objects are
+stored in the database. In particular, the various query methods should
+delegate to the Story model class. This will enable the story
+storage model to be changed without affecting this module and others above it.
+"""
+
+import copy
+
+from core.domain import story_domain
+from core.platform import models
+import feconf
+
+(story_models,) = models.Registry.import_models([models.NAMES.story])
+datastore_services = models.Registry.import_datastore_services()
+memcache_services = models.Registry.import_memcache_services()
+
+# This takes additional 'title' parameters.
+CMD_CREATE_NEW = 'create_new'
+
+# Name for the story search index.
```

Comment:
Not sure we'll have one? Don't implement unless needed. If we're not supporting search functionality for stories then drop all search functionality; we can add it later if necessary (but I don't think we'll need it since stories are bound to topics from the user's perspective).

Reply:
Done, removed it.